src/test/regress/expected/multi_mx_alter_distributed_table.out

```diff
@@ -159,5 +160,317 @@ SELECT table_name, citus_table_type, distribution_column, shard_count FROM publi
  adt_table  | distributed      | a                   |           6
 (1 row)
 
+-- test procedure colocation is preserved with alter_distributed_table
+CREATE TABLE test_proc_colocation_0 (a float8);
+SELECT create_distributed_table('test_proc_colocation_0', 'a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE OR REPLACE procedure proc_0(dist_key float8)
+LANGUAGE plpgsql
+AS $$
+DECLARE
+    res INT := 0;
+BEGIN
+    INSERT INTO test_proc_colocation_0 VALUES (dist_key);
+    SELECT count(*) INTO res FROM test_proc_colocation_0;
+    RAISE NOTICE 'Res: %', res;
+    COMMIT;
+END;$$;
+SELECT create_distributed_function('proc_0(float8)', 'dist_key', 'test_proc_colocation_0' );
+ create_distributed_function
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT logicalrelid, colocationid FROM pg_dist_partition WHERE logicalrelid::regclass::text IN ('test_proc_colocation_0');
+      logicalrelid      | colocationid
+---------------------------------------------------------------------
+ test_proc_colocation_0 |      1390028
```

Comment:
how predictable is this colocation ID? If we change other tests, would we see the same colocation ID? We may need an ALTER SEQUENCE if not.

Reply:
Makes sense, adding ALTER SEQUENCE